src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -103,7 +103,7 @@ class ModelValidationTest
       case f: Failure =>
         val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
         errors should have size 1
-        (errors.head \ "path").as[String] should be("/groups(0)/apps(1)")
+        (errors.head \ "path").as[String] should be("/groups(group1)/apps(invalid)")
```

Comment:
@pierlo-upitup If we have a sequence of things, than we usually reference this item by index like `apps(0)`. We have some logic that does not work very well with the index e.g. the apps of a group is currently a `Set` so the order is not maintained. Using an index on such structure does not help. Thats why the proposal is to identify this element by id. The proposal you see here is a relative id: `/groups(group1)/apps(invalid)` means that in group `group1` there is an app `invalid`. The full path of the app would be `/group1/invalid`. We could also use the full id instead of the relative one.